pkg/guard/manager_sql.go

```diff
@@ -15,10 +15,7 @@ import (
 )
 
 const (
-	tablePolicies  = "guard_policies"
-	tableSubjects  = "guard_subjects"
-	tableResources = "guard_resources"
-	tableActions   = "guard_actions"
+	tableName = "guard_policies_new"
```

Comment:
I suggest to stick with `guard_policies` and provide a migration script which transfers the data